### PR TITLE
resturture js code and enhence ui

### PR DESCRIPTION
**Pull request by Harryharries**

### CODE REVIEW

**Comment by jadasc** [comment]
Great job Harry! I like the idea of having descriptions for each of the pages :+1: 

The changing, blinking text is a very eye catching visual, which makes for a great addition to the home page where we have multiple statements we want to communicate. For the other pages, it might be better to keep the descriptions down to one sentence and make them static. That way they're easily accessible and people can read them at their own pace when navigating through the website (like what you did for the **Members** page)

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
+{
```
Comment (jadasc) [inline package-lock.json:1]: Since we're using Bundler, I don't think we need this file 💎

Comment (Harryharries) [reply package-lock.json]: Great suggestions! All have been changed!

**Comment by jadasc** [comment]
As we are transitioning to a new website team (yay Junhee!! 🥳 ), I think these descriptions might need to be discussed with the new team members. The descriptions should reflect the tone the committee would like to portray :)

**Pull request closed without merging**